bin/faucet/src/server/pow.rs

```diff
@@ -16,10 +19,10 @@ use super::{
     get_tokens::{InvalidRequest, RawMintRequest},
 };
 
-/// The difficulty of the `PoW`.
+/// The maximum difficulty of the `PoW`.
 ///
 /// The difficulty is the number of leading zeros in the hash of the seed and the solution.
-const DIFFICULTY: u64 = 5;
+const MAX_DIFFICULTY: u64 = 24;
```

Comment:
Related to my previous comment, since the max queue length is 1000 (AFAIK, maybe something changed or I'm missing something else), I think the effective max difficulty would be 20 here.